### PR TITLE
[New] Cart Workflow - ICDC Bentofication 

### PR DESCRIPTION
**Pull request by iksheth**
1) Added external icon property for cart custodian
2) Modified Style and GraphQL queries to get files from list
3) updated cart controller to support ICDC's uuid instead of Bento's fileId property
4) Updated cartIcon in Navbar

### CODE REVIEW

File: src/themes/icdcLight.js [hunk 1]
```diff
@@ -1,394 +0,0 @@
-import tinycolor from 'tinycolor2';
```
Comment (AjayDoddapaneni) [inline src/themes/icdcLight.js:1]: Are we removing this file?

Comment (iksheth) [reply src/themes/icdcLight.js:1]: No we are not. Its is missing in this PR. Will add and update the pr.

Comment (iksheth) [reply src/themes/icdcLight.js:1]: icdcLight was initially part of old PR. Which was separated to different PR and merged to "bento". Will update this PR accordingly.

**Review by jonkiky — APPROVED** [review]
LGTM!

**Review by skiran86 — APPROVED** [review]

**Comment by skiran86** [comment]
@iksheth Conflicts

**Pull request merged**